### PR TITLE
cap environment deployments

### PR DESCRIPTION
**Pull request by drh-stanford**

### CODE REVIEW

File: config/deploy/production.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+server 'sul-robots1-prod.stanford.edu', user: 'lyberadmin', roles: %w{web app db}
+server 'sul-robots2-prod.stanford.edu', user: 'lyberadmin', roles: %w{web app db}
```
Comment (cbeer) [inline config/deploy/production.rb]: Just checking -- we're only interested in deployed to these two servers, and not any of the others?

**Comment by drh-stanford** [comment]
Ok, I looked to see where it was deployed currently. In test it's on 1 and 5 and in prod it's on 3 and in dev it's on 1 and 2.

**Comment by cbeer** [comment]
:ok_hand: ought we deploy the robots to the other machines?

**Comment by drh-stanford** [comment]
i doesn't look like it -- 3-prod has different mounts. it needs `/web-archiving-stage`.  we probably should have a was-robots1-prod as it's confusing that sul-robots[1-5]-prod aren't identical setups.

**Comment by cbeer** [comment]
:+1:

**Pull request merged**